handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -1298,6 +1301,9 @@ private HandshakeStatus handshake() throws SSLException {
 
         // Adding the OpenSslEngine to the OpenSslEngineMap so it can be used in the AbstractCertificateVerifier.
         engineMap.add(this);
+        if (lastAccessed == -1) {
```

Comment:
the [javadocs for SSLSession.html#getLastAccessedTime](https://docs.oracle.com/javase/8/docs/api/javax/net/ssl/SSLSession.html#getLastAccessedTime--) are a bit vague but if a renegotiation is done ... do we need to set `lastAccessed`?